README.md

```diff
@@ -48,7 +48,7 @@ The current version of the plugin supports next commands:
 
  - **`:CMake`** creates a build directory (if it is necessary) and generates cmake project.
```

Comment:
Could you update the document: https://github.com/ilyachur/cmake4vim/blob/master/doc/cmake4vim.txt

Reply:
Oh yeah, I wanted to ask you, is that document automatically generated or written by hand?

Reply:
Yes, this document is written manually 😄 I don't know the way to do it automatically.